src/Feersum.CompilerServices/Compile/Compiler.fs

```diff
@@ -43,22 +63,26 @@ type CompileResult =
 module private Utils =
 
     /// Get get the size of environment object required to hold any captured values
-    /// in `envMappings`. The return from this has three main meanings:
+    /// in `envMappings`. The return from this has two main meanings:
     ///
```

Comment:
Maybe this stuff, and some of the environment logic and types from the rest of the file, can be pulled into a separate capture environments / closures